README.md

```diff
@@ -1,14 +1,15 @@
 # "Awesome" Android OEM issues (Common problems with specific Android device types from OEMs)
 
-As Android developers, we struggle frequently with the variety of Original Equipment Manufacturers(OEMs). Besides their differences in hardware, many companies use their own software overlays on open source Android OS. For Android app developers, these variations lead to unexpected bugs. Something that seems to work with most of the devices, may crash or behave in an unexpected way on a particular brand or model. And usually we recognize this rather late, through unhappy clients.
+For us Android application developers, one of our worst nightmares is to deal with unexpected issues due to the variety of Original Equipment Manufacturers(OEMs). Issues usually reported by unhappy clients on the PlayStore, because we can't test our apps on every existing Android overlays.
 
-This repository is created to share the problems we encountered with specific brands and models, so that others can look up and be cautious about it before putting their app into production. It will also provide some space for discussing possible solutions or workarounds to the problem with other developers. 
+This repository has been created to share problems we encountered with specific brands and models, to be warned about them before starting a new feature and to discuss about solutions or workarounds.
```

Comment:
```suggestion
This repository has been created to share problems we encountered with specific brands and models, so that we can be aware of them before starting a new feature and to discuss solutions or workarounds.
```